samples.json

```diff
@@ -1,3498 +1,7495 @@
 {
-  "Test": {
-    "groups": ["Favorites"],
-    "title": "Test",
-    "description": "Test",
-    "unit": "2",
-    "pixelsPerUnit": "1",
-    "pixelsPerMeter": "1000",
```

Comment:
While implementing d232978749e619cd9aed8b61013778c2a2c7de04, it took me forever to figure out why the grid size was changing when I dropped `unit` and `pixelsPerUnit`. 😭 I finally noticed that this `Test` sample had inconsistent units. The unit is set to microns, and pixels per meter is set to 1000, which should indicate pixels per unit (i.e. micron) = 0.001, but instead it's 1. I changed `pixelsPerMeter` to `1000000` to conform to `pixelsPerUnit`.

This illustrates why it's good not to store redundant fields - they can be inconsistent!